src/js/smooch.jsx

```diff
@@ -294,13 +294,16 @@ export class Smooch {
             console.warn('Smooch.destroy was called before Smooch.init was called properly.');
         }
 
-        const {embedded} = store.getState().appState;
-        disconnectFaye();
-        store.dispatch(reset());
+        stopMonitoringBrowserState();
+        
         if (process.env.NODE_ENV !== 'test' && this._container) {
             unmountComponentAtNode(this._container);
         }
 
+        disconnectFaye();
+        store.dispatch(reset());
+
+        const {embedded} = store.getState().appState;
```

Comment:
I guess this state will still be correct even though you're resetting right before?